apps/api/src/app/subscribers/usecases/index.ts

```diff
@@ -2,5 +2,16 @@ import { CreateSubscriber } from './create-subscriber';
 import { UpdateSubscriber } from './update-subscriber';
 import { RemoveSubscriber } from './remove-subscriber';
 import { GetSubscribers } from './get-subscribers';
+import { GetSubscriberPreference } from '../../widgets/usecases/get-subscriber-preference/get-subscriber-preference.usecase';
+import { UpdateSubscriberPreference } from '../../widgets/usecases/update-subscriber-preference/update-subscriber-preference.usecase';
+import { GetSubscriberTemplatePreference } from '../../widgets/usecases/get-subscriber-template-preference';
 
-export const USE_CASES = [CreateSubscriber, UpdateSubscriber, RemoveSubscriber, GetSubscribers];
+export const USE_CASES = [
+  CreateSubscriber,
+  UpdateSubscriber,
+  RemoveSubscriber,
+  GetSubscribers,
+  GetSubscriberPreference,
+  UpdateSubscriberPreference,
+  GetSubscriberTemplatePreference,
```

Comment:
I couldnt import the widget module to this one due to the circular dependence issue. 
Is this approach valid? (i am asking because haven't seen any similar imports from other modules)

Reply:
Not really, it breaks the module boundaries and can bite us later. In my opinion, we can have those usecases in the subscriber module rather than in the widget module, and than you won't have circular dependencies. Circular dependency usual means there is a better way to structure the code :)